wandb/sdk/internal/internal_api.py

```diff
@@ -2180,6 +2182,9 @@ def no_retry_4xx(e: Exception) -> bool:
             except UsageError as e:
                 raise e
             except Exception as e:
+                wandb.termwarn(
```

Comment:
termwarn is too noisy for this type of message,  we are retrying it so the user should never care.
if anything the error would need to be reported back to the caller and your launch sweep controller would be responsible for telling the user that this feature is not available because their wandb server is too old